pom.xml

```diff
@@ -8,7 +8,7 @@
     <groupId>org.jboss.integration-platform</groupId>
     <artifactId>jboss-integration-platform-bom</artifactId>
     <!-- Keep in sync with ip-parent version in kie-user-bom-parent/pom.xml -->
-    <version>8.0.0.Final</version>
+    <version>8.1.0-SNAPSHOT</version>
```

Comment:
@porcelli @mbiarnes same case of errai one. How I should proceed in order to make the build green without this?